WordPress/WordPressShareExtension/ShareViewController.swift

```diff
@@ -251,19 +295,54 @@ private extension ShareViewController {
     }
 }
 
-
-
 /// ShareViewController Extension: Backend Interaction
 ///
 private extension ShareViewController {
     func uploadPostWithSubject(_ subject: String, body: String, status: String, siteID: Int, attachedImageData: Data?, requestEqueued: @escaping () -> ()) {
-        let configuration = URLSessionConfiguration.backgroundSessionConfigurationWithRandomizedIdentifier()
-        let service = PostService(configuration: configuration)
 
-        service.createPost(siteID: siteID, status: status, title: subject, body: body, attachedImageJPEGData: attachedImageData, requestEqueued: {
+        guard let attachedImageData = attachedImageData else { return }
```

Comment:
You could join this two guard in a single one, and according to the new code style guide the return needs to be in a new line.